pkgs/development/libraries/libgit2/default.nix

```diff
@@ -46,7 +46,23 @@ stdenv.mkDerivation rec {
 
   propagatedBuildInputs = lib.optional (!stdenv.isLinux) libiconv;
 
-  doCheck = false; # hangs. or very expensive?
+  doCheck = true;
+  checkPhase = ''
+    testArgs=(-v -xonline)
+
+    # slow
+    testArgs+=(-xclone::nonetwork::bad_urls)
+
+    # failed to set permissions on ...: Operation not permitted
+    testArgs+=(-xrepo::init::extended_1)
+    testArgs+=(-xrepo::template::extended_with_template_and_shared_mode)
+
+    ( set -x
+      ./libgit2_tests ''${testArgs[@]}
+    )
+  '';
+
+  outputs = ["lib" "dev" "out"];
```

Comment:
Usually we place outputs near the top after version and before src. Also some extra spaces inside the square brackets does not make it so tight and cramped.